src/backend/funmooc/base/tests/test_settings.py

```diff
@@ -0,0 +1,22 @@
+"""Test fun-mooc configuration."""
+from unittest import mock
+
+from django.test import TestCase
+
+from rest_framework.response import Response
+from richie.apps.search.viewsets.courses import CoursesViewSet
+
+
+class ConfigurationTestCase(TestCase):
+    """Validate that our configuration works as expected."""
+
+    def test_configuration_restframework_htaccess(self):
+        """The search API endpoint should work behind an htaccess."""
+        with mock.patch.object(
+            CoursesViewSet, "list", spec=True, return_value=Response({})
+        ):
+            response = self.client.get(
+                "/api/v1.0/courses/",
+                HTTP_AUTHORIZATION="Basic dXNlcm5hbWU6cGFzc3dvcmQ=",
+            )
+        self.assertEqual(response.status_code, 200)
```

Comment:
I have the feeling that this test is quite weak: 

1. IMO we need to test at least two conditions, _i.e._ with and without basic auth? 
2. Mocking the response you are testing and be happy that the mock returns a 200 response is... surprising! Am i missing something?
3. Where does the HTTP Basic auth token comes from?

Reply:
I don't think the test was weak but the explanation was. I added a counter test as you suggested and tried to explain the problem better in the test and in the PR description.